src/Command/HistoryCommand.php

```diff
@@ -176,13 +176,13 @@ private function extractRange(string $range): array
     /**
      * Retrieve a slice of the readline history.
      *
-     * @param string $show
-     * @param string $head
-     * @param string $tail
+     * @param string|null $show
+     * @param string|null $head
+     * @param string|null $tail
      *
-     * @return array A slilce of history
+     * @return array A slice of history
      */
-    private function getHistorySlice(string $show, string $head, string $tail): array
+    private function getHistorySlice(?string $show, ?string $head, ?string $tail): array
```

Comment:
PsySH supports PHP 7.0, so we need to remove the type hints rather than marking them nullable.

Reply:
I'll remove the typehints then 👍

Reply:
I've removed the typehints. I left the docblock update, to avoid someone accidently reintroducing the same typehints in the future